staging/src/github.com/kubeedge/mapper-framework/_template/mapper/device/device.go

```diff
@@ -131,7 +131,7 @@ func dataHandler(ctx context.Context, dev *driver.CustomizedDev) {
 			ObservedDesired: twin.ObservedDesired,
 			VisitorConfig:   &visitorConfig,
 			Topic:           fmt.Sprintf(common.TopicTwinUpdate, dev.Instance.ID),
-			CollectCycle:    time.Duration(twin.Property.CollectCycle),
+			CollectCycle:    time.Millisecond * time.Duration(twin.Property.CollectCycle),
```

Comment:
Thanks for the fixing, and the unit here is ms or second? Any comments or docs?

Reply:
Thank you very much for pointing out the problem. I have added the explanation in the document and also modified the time unit of the reporting cycle.